File: Exercicio5.c
#include <stdio.h>
#include <stdlib.h>
#include <locale.h>

int main()
{
    setlocale(LC_ALL,"");
    int cp1, qnt1, cp2, qnt2;
    float valor1, valor2, total;

    printf("Digite o código da primeira peça: ");
    scanf("%d",&cp1);
    printf("\nDigite a quantidade da peça 1: ");
    scanf("%d",&qnt1);
    printf("\nDigie o valor unitário da peça 1: ");
    scanf("%f",&valor1);
    printf("\nDigite o código da segunda peça: ");
    scanf("%d",&cp2);
    printf("\nDigite a quantidade da peça 2: ");
    scanf("%d",&qnt2);
    printf("\nDigie o valor unitário da peça 2: ");
    scanf("%f",&valor2);

    total = (qnt1*valor1)+(qnt2*valor2);

    printf("\n\nO código e valor da primeira peça são %d e %f \nO código e valor da segunda peça são %d e %f \n",cp1,valor1,cp2,valor2);
    printf("O valor da compra é %f \n",total);
    return 0;
}

File: Exercicio5.cs
using System;

class Program {
  public static void Main (string[] args)
        {
            int cp1, qnt1, cp2, qnt2;
            float valor1, valor2, total;

            Console.WriteLine("Digite o codigo da primeira peca: ");
            cp1 = int.Parse(Console.ReadLine());
            Console.WriteLine("Digite a quantidade da peca 1: ");
            qnt1 = int.Parse(Console.ReadLine());
            Console.WriteLine("Digite o valor unitario da peca 1: ");
            valor1 = float.Parse(Console.ReadLine());
            Console.WriteLine("Digite o codigo da segunda peca: ");
            cp2 = int.Parse(Console.ReadLine());
            Console.WriteLine("Digite a quantidade da peca 2: ");
            qnt2 = int.Parse(Console.ReadLine());
            Console.WriteLine("Digite o valor unitario da peca 2: ");
            valor2 = float.Parse(Console.ReadLine());

            total = (qnt1 * valor1) + (qnt2 * valor2);
            Console.WriteLine("\r\n O codigo e valor da primeira peca são " + cp1 + " e " + valor1);
            Console.WriteLine(" O codigo e valor da segunda peca são " + cp2 + " e " + valor2);
            Console.WriteLine("\r\n O valor da compra eh " + total);

        }
}